src/block/block_read.c

```diff
@@ -13,15 +13,17 @@
  *     Map or read address cookie referenced block into a buffer.
  */
 int
-__wt_bm_read(
-  WT_BM *bm, WT_SESSION_IMPL *session, WT_ITEM *buf, const uint8_t *addr, size_t addr_size)
+__wt_bm_read(WT_BM *bm, WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *block_meta,
+  const uint8_t *addr, size_t addr_size)
 {
     WT_BLOCK *block;
     WT_DECL_RET;
     wt_off_t offset;
     uint32_t checksum, objectid, size;
     bool last_release;
 
+    WT_UNUSED(block_meta);
```

Comment:
Why did you add this argument if it's unused? Is it for future or an overlook?

Reply:
Good question! This is here because the block manager is an interface. The disagg block manager needs this extra argument, so now the existing block manager has to discard it. (Same deal in `block_write.c`.)